src/Metadata/HttpOperation.php

```diff
@@ -18,14 +18,6 @@
 
 class HttpOperation extends Operation
 {
-    public const METHOD_GET = 'GET';
-    public const METHOD_POST = 'POST';
-    public const METHOD_PUT = 'PUT';
-    public const METHOD_PATCH = 'PATCH';
-    public const METHOD_DELETE = 'DELETE';
-    public const METHOD_HEAD = 'HEAD';
-    public const METHOD_OPTIONS = 'OPTIONS';
```

Comment:
Since we want to use raw string, I don't see the point of those constant to remain.
So I've deleted them

Reply:
Let's ask to @dunglas and @chalasr then, what should be done there ?

How do you wish to proceed ?

Reply:
Reasoning is: these constants are for end users, api platform does not reference them for performance reasons. They must stay, and the codebase should not use them at all for consistency as per @dunglas statement.

Reply:
I totally agree with @chalasr

Reply:
It's done then, I've reverted it back the way it was :)